source/_components/snmp.markdown

```diff
@@ -91,12 +91,12 @@ baseoid:
   description: The OID prefix where wireless client registrations can be found, usually vendor specific. It's advised to use the numerical notation. To find this base OID, check vendor documentation or check the MIB file for your device.
   required: true
   type: string
-authkey:
-  description: Authentication key for SNMPv3. Variable privkey must also be set.
+auth_key:
+  description: A"uthentication key for SNMPv3. Variable `priv_key` must also be set."
```

Comment:
The quote got misplaced inside authentication.

Reply:
new PR fot that?

Reply:
It would be good to fix it yes. Is that the question?

Reply:
yes that was the question, if it need to be adjusted then just in a new PR 🤣